frontend/packages/client/craco.config.js

```diff
@@ -0,0 +1,24 @@
+const path = require('path');
+const { getLoader, loaderByName } = require('@craco/craco');
+
+const packages = [];
+packages.push(path.join(__dirname, '../shared-components'));
+
+module.exports = {
+  webpack: {
+    configure: (webpackConfig, arg) => {
+      const { isFound, match } = getLoader(
+        webpackConfig,
+        loaderByName('babel-loader')
+      );
+      if (isFound) {
+        const include = Array.isArray(match.loader.include)
+          ? match.loader.include
+          : [match.loader.include];
+
+        match.loader.include = include.concat(packages);
+      }
+      return webpackConfig;
+    },
+  },
+};
```

Comment:
[craco](https://www.npmjs.com/package/@craco/craco) is used so we can load things outside of `packages/client` with our create-react-app babel config